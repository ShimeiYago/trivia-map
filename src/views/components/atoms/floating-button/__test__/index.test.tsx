import { shallow, ShallowWrapper } from 'enzyme';
import { FloatingButton, Props } from '..';

let wrapper: ShallowWrapper<Props, unknown, FloatingButton>;

describe('Shallow Snapshot Tests', () => {
  beforeEach(() => {
    wrapper = shallow(<FloatingButton />);
  });

  it('basic', () => {
    expect(wrapper).toMatchSnapshot();
  });

  it('edit icon', () => {
    wrapper.setProps({
      icon: 'edit',
    });
    expect(wrapper).toMatchSnapshot();
  });
});
