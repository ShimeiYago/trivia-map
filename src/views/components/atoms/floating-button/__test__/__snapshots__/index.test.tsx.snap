// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Shallow Snapshot Tests add-marker icon 1`] = `
<ForwardRef(Fab)
  color="primary"
  disabled={false}
  sx={
    Object {
      "bottom": 20,
      "left": "auto",
      "margin": 0,
      "position": "fixed",
      "right": 20,
      "top": "auto",
      "zIndex": 1000,
    }
  }
>
  <Memo(ForwardRef(AddLocationAltIcon)) />
</ForwardRef(Fab)>
`;

exports[`Shallow Snapshot Tests basic 1`] = `
<ForwardRef(Fab)
  color="primary"
  disabled={false}
  sx={
    Object {
      "bottom": 20,
      "left": "auto",
      "margin": 0,
      "position": "fixed",
      "right": 20,
      "top": "auto",
      "zIndex": 1000,
    }
  }
>
  <Memo(ForwardRef(AddIcon)) />
</ForwardRef(Fab)>
`;

exports[`Shallow Snapshot Tests edit icon 1`] = `
<ForwardRef(Fab)
  color="primary"
  disabled={false}
  sx={
    Object {
      "bottom": 20,
      "left": "auto",
      "margin": 0,
      "position": "fixed",
      "right": 20,
      "top": "auto",
      "zIndex": 1000,
    }
  }
>
  <Memo(ForwardRef(EditIcon)) />
</ForwardRef(Fab)>
`;
